webpack.config.devtools.js

```diff
@@ -28,6 +27,8 @@ module.exports = webpackConfig => {
     webpackConfig.output.path = path.join(__dirname, "firefox/devtools/client/debugger/new");
   }
 
+  webpackConfig.resolve.alias["public/js/utils/networkRequest"] = "public/js/utils/privilegedNetworkRequest";
```

Comment:
I could move it to `lib` at least. This is something that we will be working on solely for ourselves for the time being, so I don't really want want to go through m-c and complicate that process when nobody else in m-c will even use this.

But I think it makes sense to live in `lib` at least, which indicates that it's strictly for Firefox.